lib/govuk_publishing_components/presenters/public_layout_helper.rb

```diff
@@ -1,294 +1,7 @@
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Departments",
-                },
-              },
-            },
-            {
-              href: "/world",
-              text: "Worldwide",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Worldwide",
-                },
-              },
-            },
-            {
-              href: "/search/services",
-              text: "Services",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Services",
-                },
-              },
-            },
-            {
-              href: "/search/guidance-and-regulation",
-              text: "Guidance and regulation",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Guidance and regulation",
-                },
-              },
-            },
-            {
-              href: "/search/news-and-communications",
-              text: "News and communications",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "News and communications",
-                },
-              },
-            },
-            {
-              href: "/search/research-and-statistics",
-              text: "Research and statistics",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Research and statistics",
-                },
-              },
-            },
-            {
-              href: "/search/policy-papers-and-consultations",
-              text: "Policy papers and consultations",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Policy papers and consultations",
-                },
-              },
-            },
-            {
-              href: "/search/transparency-and-freedom-of-information-releases",
-              text: "Transparency and freedom of information releases",
-              attributes: {
-                data: {
-                  track_category: "footerClicked",
-                  track_action: "footerLinks",
-                  track_label: "Transparency and freedom of information releases",
-                },
-              },
-            },
-          ],
-        },
-      ].freeze
+      FOOTER_NAVIGATION_RATIO = [2, 1].freeze
```

Comment:
Is there a particular reason we need to freeze the variable here? Is it just to guard against someone changing this but not updating the data to reflect? Might be worth leaving a comment about it?

I'm also wondering if `FOOTER_NAVIGATION_COLUMNS` might be a more descriptive name? I had to look at the code to understand what it was for. But not a blocker.

Reply:
`.freeze` is for making the constant a constant. Name change suggestion seems sensible.

Reply:
Thanks @chao-xian, I got that but was querying if there was a particular scenario where it might be at risk of being overwritten.